l1-contracts/test/Rollup.t.sol

```diff
@@ -116,6 +123,15 @@ contract RollupTest is DecoderBase {
     vm.warp(Timestamp.unwrap(rollup.getTimestampForSlot(Slot.wrap(_slot))));
   }
 
+  function skipBlobCheck() public {
```

Comment:
Is this necessary, wcould you not use the `checkBlob` value that you added in storage to turn it of or on?

Reply:
This is historical - the `checkBlob` came much after adding the above. Will make the test use `checkBlob` instead 👍